airflow/utils/log/es_task_handler.py

```diff
@@ -262,3 +267,25 @@ def close(self):
         super().close()
 
         self.closed = True
+
+    @property
+    def log_name(self):
+        return self.LOG_NAME
+
+    def get_external_log_url(self, task_instance: TaskInstance, try_number: int) -> str:
+        """
+        Creates an address for an external log collecting service.
+        :param task_instance: task instance object
```

Comment:
```suggestion

        :param task_instance: task instance object
```
Without this, the documentation does not comply with the Sphinx.